recipes/bio_hansel/meta.yaml

```diff
@@ -31,6 +32,7 @@ requirements:
     - numpy >=1.12.1
     - pandas >=0.20.1
     - attrs ==17.2.0
+    - pyahocorasick >=1.1.6
```

Comment:
This seems to be missing in our supported channels.

Reply:
Should be available on `conda-forge` now.

Reply:
Ah, this package is not available for OSX. I added `[not py3k]` to the skip parameter.